src/kv/store.h

```diff
@@ -509,27 +537,17 @@ namespace kv
           return DeserialiseSuccess::FAILED;
         }
 
-        // if we are not committing now then use NoVersion to deserialise
+        // If we are not committing now then use NoVersion to deserialise
         // otherwise the view will be considered as having a committed
         // version
         auto deserialise_version = (commit ? v : NoVersion);
-        auto deserialised_write_set =
+        auto deserialised_view =
```

Comment:
Small changes here: AIUI, `deserialise()` returns a view and not a write set (rename) and which can never be `nullptr` (removing error path below).